health-services/project-factory/src/server/api/campaignApis.ts

```
@@ -171,132 +239,255 @@
   newCreatedData.forEach((element: any) => {
     delete element[uid];
-  })
-  var errors: any[] = []
+  });
+  var errors: any[] = [];
   if (request?.body?.ResourceDetails?.type != "user") {
     if (request?.body?.ResourceDetails?.type == "facility") {
       newCreatedData?.forEach((element: any) => {
-        delete element.address
-      })
+        delete element.address;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)